www/apps/book/app/advanced-development/api-routes/protected-routes/page.mdx

```diff
@@ -151,8 +163,8 @@ export const config: MiddlewaresConfig = {
 
 The `authenticate` middleware function accepts three parameters:
 
-1. The scope of authentication. Use `admin` for authenticating admin users, and `store` for authenticating customers.
-2. An array of the types of authentication methods allowed. Both `admin` and `store` scopes support `session` and `bearer`. The `admin` scope also supports the `api-key` authentication method.
+1. The scope of authentication. Use `user` for authenticating admin users, and `customer` for authenticating customers.
```

Comment:
We moved away from using `scope` (as that usually means something different in the authz world) to using `Actor type`